app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ExpandedArticleFragment.java

```diff
@@ -0,0 +1,115 @@
+package edu.grinnell.grinnell_publications_android.Fragments;
+
+
+import android.graphics.drawable.Drawable;
+import android.support.annotation.Nullable;
+import android.os.Bundle;
+
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
+
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.UserInterface;
+import edu.grinnell.grinnell_publications_android.R;
+
+
+/**
+ * Fragment that displays the expanded article content.
+ * Fragment allows for favoriting of the article.
+ */
+
+public class ExpandedArticleFragment extends Fragment implements UserInterface {
+
+    private ImageView mHeaderImage;
+    private CollapsingToolbarLayout mCollapsingToolbar;
+    private FloatingActionButton mFavoriteButton;
+    private TextView mArticleContent;
+    private Toolbar mArticleToolbar;
```

Comment:
Please destroy all of these member variables in Fragment's `onDestroyView()` as it allows the fragment to clean up resources associated with its View. That way, we're not leaking any memory references on the Fragment's destruction.